### PR TITLE
Add more stats intercom

### PR DESCRIPTION
**Pull request by tspenov**
Intercom custom attributes:
* used_sanapi
* used_sansheets
* api_calls_count
* weekly_report_watchlist_count

### CODE REVIEW

File: lib/sanbase/clickhouse/api_call_data.ex [hunk 1]
```diff
@@ -31,6 +32,38 @@ defmodule Sanbase.Clickhouse.ApiCallData do
...
+      {:error, _error} -> []
+    end
+  end
+
+  def users_used_sansheets() do
+    {query, args} = users_used_sansheets_query()
+
+    ClickhouseRepo.query_transform(query, args, fn value -> value end)
+    |> case do
+      {:ok, result} -> List.flatten(result)
```
Comment (IvanIvanoff) [inline lib/sanbase/clickhouse/api_call_data.ex]: this is doing nothing

File: lib/sanbase/clickhouse/api_call_data.ex [hunk 2]
```diff
@@ -31,6 +32,38 @@ defmodule Sanbase.Clickhouse.ApiCallData do
...
+  end
+
+  def api_calls_count_per_user() do
+    {query, args} = api_calls_count_per_user_query()
+
+    ClickhouseRepo.query_transform(query, args, fn [user_id, count] ->
+      {user_id, String.to_integer(count)}
+    end)
+    |> case do
+      {:ok, result} -> result |> Enum.into(%{})
```
Comment (IvanIvanoff) [inline lib/sanbase/clickhouse/api_call_data.ex]: you can use `query_reduce` and directly put them into the map

File: lib/sanbase/clickhouse/api_call_data.ex (hunk 1 above)
Comment (tspenov) [reply lib/sanbase/clickhouse/api_call_data.ex]: it returns list of : `[[1], [2]]`, so flatten it

Comment (tspenov) [reply lib/sanbase/clickhouse/api_call_data.ex]: flattened during transform

**Pull request merged**